project-communications/newsletter/newsletter-012921.md

```diff
@@ -0,0 +1,180 @@
+---
+breaks: False
+---
+# Newsletter | 29 January 2020
+
+*Date sent: 01 February 2020*
+
+Hello, 
+
+We're already one month into 2021! We hope your year has had a good beginning. 
```

Comment:
```suggestion
We're already one month into 2021!
We hope your year has had a good beginning. 
```

Reply:
Can you put all of the newsletter with each sentence on a new line @GeorgiaHCA? Ping me on slack if I can help.

Reply:
Oh wait - I think this was just a missed one at the top... reading down now.

Reply:
Yep, sorry! Missed that one! Guess because they were super short sentences

Reply:
Thanks for finding it :)